app/views/discovered_hosts/show.html.erb

```diff
@@ -1,16 +1,17 @@
-<% title(_("Discovered host: %s") % @host.to_label) %>
-
-<%= discovered_hosts_title_actions(@host) %>
```

Comment:
should the action buttons still exist in show?
they are all present in the index